### DISCUSSION TITLE
Blackhole recognised as malware after upgrading macOS to 11.3.1

### DISCUSSION THREAD

**Discussion by allhandsondeck** [ExistentialAudio/BlackHole#388] [Q&A] [answered]
Hello,

I have upgraded macOS to 11.3.1 and after upgrade finished and reboot, macOS showed me a warning that BlackHole is a malware.
Same warning had a button redirecting to HAL folder.
There I see two drivers: blackhole.driver and blackhole2ch.driver. Eventually, I deleted both of them.
Is that a known issue? Are both those 2 drivers belong to the existentialAudio/BlackHole?

Thanks in advance.

**Comment by devinroth** [comment, marked as answer]
Yes both are correct. I’ve heard a few people notify me about this. Might be from having an older version. Try the latest and see if it makes a difference.

Devin Roth

**Reply by allhandsondeck** [reply]
Will do that.

Thanks!

**Reply by Paul-Woodcraft** [reply]
Downloaded a new copy 2 Ch v02.9. Went through the instalation process but no warnings & the application doesn't appear to be installed.
It is listed in Audio MIDI setup but that may be legacy from previous use rather than available to use now.
However in System Preference > Security&Privacy > Microphone - BlackHole is no longer listed
![BlackHole](https://user-images.githubusercontent.com/25765893/117578417-2c789a00-b0e6-11eb-94b4-36eba6731c8b.png)

**Reply by Madhawa97** [reply]
I'm on 11.3. I have installed BlackHole (Latest version) and its not listed under the microphone access. its on the Audio Midi setup and Works as expected.

**Reply by Paul-Woodcraft** [reply]
Thanks

**Comment by devinroth** [comment]
If it’s in Audio Midi setup it’s correctly installed.

Please read setup guides and tutorials in order to learn how to use it.

Devin Roth
